components/Members/MemberItem.tsx

```diff
@@ -42,48 +41,25 @@ const MemberItem = ({ item }: { item: Member }) => {
         ])
       : null;
 
+  const walletAddressFormatted = abbreviateAddress(
+    walletPublicKey as PublicKey,
```

Comment:
If you don't handle the potential `undefined` returned by `tryParsePublicKey`:

- Either we assume the wallet address is always a valid public key, and then we should use:

```
   const walletPublicKey = new PublicKey(walletAddress);
```

instead of

```
const walletPublicKey = tryParsePublicKey(walletAddress);
```

- Either we accept that the wallet address may not be a correct public key and chose a specific display:

```
 const walletAddressFormatted = walletPublicKey ? abbreviateAddress(
    walletPublicKey,
  ) : '-';
```

instead of 

```
 const walletAddressFormatted = abbreviateAddress(
    walletPublicKey as PublicKey,
  );
```